src/layouts/Settings.jsx

```diff
@@ -104,30 +104,41 @@ export default class Settings extends Component {
         configFieldsRequired,
         footerContent,
         navigationContent,
-        footerSha,
       } = settings;
 
-      // set state properly
-      if (this._isMounted) this.setState((currState) => ({
-        ...currState,
-        siteName,
-        ...configFieldsRequired,
+      const originalState = {
+        // config fields
+        // resources_name: configFieldsRequired.resources_name,
+        // url: configFieldsRequired.url,
+        colors: configFieldsRequired.colors,
+        favicon: configFieldsRequired.favicon,
+        google_analytics: configFieldsRequired.google_analytics,
+        facebook_pixel: configFieldsRequired.facebook_pixel,
+        is_government: configFieldsRequired.is_government,
+        shareicon: configFieldsRequired.shareicon,
+        title: configFieldsRequired.title,
```

Comment:
Might be helpful to leave a comment that we list out config fields so we only retrieve those which are used in the page, otherwise it causes trouble when getting the diff

Reply:
Done with [5db6a82](https://github.com/isomerpages/isomercms-frontend/pull/285/commits/5db6a8289a88020282c8d87b85b661a0603a7548)